terraform/deployments/cluster-services/main.tf

```diff
@@ -10,7 +10,7 @@ terraform {
   cloud {
     organization = "govuk"
     workspaces {
-      tags = ["cluster-services", "eks", "aws"]
+      tags = ["cluster-services", "aws"]
```

Comment:
Not sure how this change made it in, but it fixes a small issue where the Terraform CLI doesn't work for this workspace in ephemeral environments (tags aren't set correctly)